src/main/java/org/mqttbee/mqtt/MqttClientSslConfigImpl.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2018 The MQTT Bee project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mqttbee.mqtt;
+
+import org.mqttbee.annotations.NotNull;
+import org.mqttbee.annotations.Nullable;
+import org.mqttbee.api.mqtt.MqttClientSslConfig;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.TrustManagerFactory;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ *  Default MqttClientSslConfig implementation:
+ *
+ *  Handshake timeout set to 10 seconds and pre-selected
+ *  list of allowed cipher suites
+ *
+ *  @author David Katz
+ */
+public class MqttClientSslConfigImpl implements MqttClientSslConfig {
+    public static final MqttClientSslConfig DEFAULT = new MqttClientSslConfigImpl(null, null);
+    private static final List<String> cipherSuites = Arrays.asList("TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
```

Comment:
ImmutableList<String> DEFAULT_CIPHER_SUITES

Reply:
Imho we should use the platform's default cipher suites instead of choosing them here and give the user the possibility to specify them in the builder?

Reply:
These cipher suites are all pretty good but still very new.
So as the default we would be very restrictive.

Reply:
Netty uses a set of defualt cipher suites https://github.com/netty/netty/blob/268b901844c620337bc11ae82e99163a1781b96c/handler/src/main/java/io/netty/handler/ssl/SslUtils.java#L87

These differ from the ones that are define here.

I will switch to netty defaults. Or do you have any complains?

Reply:
DONE